platform/ios/src/MGLMapView.h

```diff
@@ -1183,10 +1202,11 @@ IB_DESIGNABLE
  @param styleLayerIdentifiers A set of strings that correspond to the names of
     layers defined in the current style. Only the features contained in these
     layers are included in the returned array.
+ @param predicate A predicate to filter the returned features.
  @return An array of objects conforming to the `MGLFeature` protocol that
     represent features in the sources used by the current style.
  */
-- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(nullable NS_SET_OF(NSString *) *)styleLayerIdentifiers NS_SWIFT_NAME(visibleFeatures(at:styleLayerIdentifiers:));
+- (NS_ARRAY_OF(id <MGLFeature>) *)visibleFeaturesAtPoint:(CGPoint)point inStyleLayersWithIdentifiers:(nullable NS_SET_OF(NSString *) *)styleLayerIdentifiers predicate:(nullable NSPredicate*)predicate NS_SWIFT_NAME(visibleFeatures(at:styleLayerIdentifiers:predicate:));
```

Comment:
Nit: space between `NSPredicate` and `*`. The predominent Objective-C coding style is `Type *name` and `Type * const name`. There’s less certainty about the proper style for Objective-C++ code, which is one reason we’ve yet to lint for that sort of stylistic detail.